arches/app/models/tile.py

```diff
@@ -579,16 +579,15 @@ def is_blank(self):
 
     @staticmethod
     def get_blank_tile(nodeid, resourceid=None):
-        parent_nodegroup = None
         node = models.Node.objects.get(pk=nodeid)
```

Comment:
should we be using `prefetch_related('nodegroup')` here?

Reply:
That's a good suggestion, but prefetch_related is a method on a QuerySet. When we call `objects.get` we don't get a QuerySet, but rather the Node object itself which doesn't have the `prefetch_related` method. 

We could instead do something like:
```models.Node.objects.filter(pk=nodeid).prefetch_related('nodegroup')[0]```

Reply:
I've updated the PR to use `select_related`.  It turns out `prefetch_related` would still used 2 queries joining the data with python. `select_related` uses a single query with a JOIN.